### PR TITLE
Cindy / implement functions to collect networking metrics

### PR DESCRIPTION
**Pull request by cindyariyo**
Three functions returning inbound traffic, outbound traffic and a string representation of these metrics.

This PR cloes #48.

### CODE REVIEW

File: exporter/collectors/network.py [hunk 1]
```diff
@@ -2,30 +2,50 @@
...
         TODO:
         Returns to string representation of all networking metrics.        
         """
-        pass
+       
+        traffic_in = self.get_traffic_in()
+        traffic_out = self.get_traffic_out()
+        return f"Inbound Traffic: {traffic_in} Mb/s, Outbound Traffic: {traffic_out} Mb/s"
+
+network_collector = NetworkCollector()
```
Comment (ljdzed) [inline exporter/collectors/network.py:49]: **Requested Changes**
It seems you've left some of your code from testing in this file. The collector files should not print anything out themselves. Code in Question:

> network_collector = NetworkCollector()
> print(network_collector.__str__())

**Comment by max-romagnoli** [comment]
@ljdzed attention needed on this PR.
Let's try to close it soon, adding at least one or two tests and seeing if Prometheus reads the metrics exposed.

File: tests/exporter/collectors/test_network.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import unittest
+from exporter.collectors.network import NetworkCollector
+
+class TestNetworkCollector(unittest.TestCase):
```
Comment (max-romagnoli) [inline tests/exporter/collectors/test_network.py:4]: ### Requested Changes
* Add the following two tests for the Traffic In and Traffic Out:
  * `test_get_traffic_in_is_not_negative` :  check if method returns >= 0
  * `test_get_traffic_out_is_not_negative` : check if method returns >= 0

**Review by max-romagnoli — CHANGES_REQUESTED** [review]
@cindyariyo good job with the methods and tests.
I have tested it with Prometheus and the metrics are returned correctly!
I also merged the `dev` into yours to solve any conflict.

You can find comments on two little requested changes. But other than that we should be able to close this today.

File: exporter/metrics.py [hunk 3]
```diff
@@ -2,4 +2,10 @@
 
 cpu_utilization_gauge = Gauge("cpu_utilization", "CPU utilization percentage")
 cpu_frequency_gauge = Gauge("cpu_frequency", "CPU frequency in MHz")
-cpu_temperature_gauge = Gauge("cpu_temperature", "CPU temperature in Celsius")
\ No newline at end of file
+cpu_temperature_gauge = Gauge("cpu_temperature", "CPU temperature in Celsius")
+
+network_get_traffic_in_gauge = Gauge("stagent_traffic_in_mbs", "NETWORK inbound traffic in Mb/s")
+network_get_traffic_out_gauge = Gauge("stagent_traffic_out_mbs", "NETWORK outbound traffic in Mb/s")
```
Comment (max-romagnoli) [inline exporter/metrics.py:8]: ### Comment
* Here I have renamed the Gauges to reflect naming conventions.

**Review by max-romagnoli — APPROVED** [review]
# LGTM

**Pull request merged**